### PR TITLE
Handlers

### PR DESCRIPTION
**Pull request by SaiKumarMalve**
Writing handlers to handle PORT listeners and DB connection in Main.go

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
-func main() {
+import (
+	"log"
+
+	"github.com/SaiKumarMalve/Gator-News/bd"
+	"github.com/SaiKumarMalve/Gator-News/handlers"
+)
 
+func main() {
+	if bd.CheckConnection() == 0 {
```
Comment (Alekhyagnv) [inline main.go:11]: wait for my changes to be merged to main

**Review by Alekhyagnv — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (Alekhyagnv) [reply main.go:11]: I merged my changes to main.
Everything looks good. Go Ahead and merge your changes.

Comment (SaiKumarMalve) [reply main.go:11]: Okay. Sure

Comment (Alekhyagnv) [reply main.go:11]: I merged my changes to main.
Everything looks good. Go Ahead and merge your changes.

**Pull request merged**